### ISSUE TITLE
blink Black screen with super-navigate

### ISSUE THREAD

**Issue by vanleab** [AppGyver/steroids#820]
i use steroids 3.5.10 and supersonic 1.1.1. on android, whenever i navigate to new layer that is not preloaded, it always show a blink black screen before the layer show. the content of the layer fetch from $http json. on ios, none of this happen.

**Comment by poppahorse** [comment]
steroids 4.0.11, same issue

**Comment by pre** [comment]
Are you using Crosswalk or the Platform WebView? Which Android devices you have encountered this problem with? Which Android OS versions they have?

**Comment by poppahorse** [comment]
Crosswalk. SGS3 + SGS4 on lollipop (both only black for a split second), and Tesco Hudl on KitKat (much larger, more noticable delay on the black bg)

**Comment by vanleab** [comment]
use preload layers. but it still show blink previous content before new content is loaded, and it is faster though

**Comment by harsaharsa** [comment]
Could this be a duplicate of https://github.com/AppGyver/steroids/issues/830 ? Are there any Web Components in the offending page?

**Comment by carlo** [comment]
I've set up preloading for my views, but the flicker remains when using `<super-navigate location="example#show"…` — yet it's gone when I use `<super-navigate view-id="example-show"…`.

Might be relevant.  I'm using Crosswalk.

**Comment by harsaharsa** [comment]
The `view-id` tag uses the preloaded view, the `location` tag creates a NEW WebView from that location string, so the behavior is as expected (i.e. the flicker only affects non-preloaded views).

**Comment by carlo** [comment]
That makes sense, @harsaharsa.  Thanks for the clarification!

**Comment by kkgelu** [comment]
@harsaharsa  @pre @AppGyver yes, it affects non-preloaded views, it only happens on Androids. It's Super-ugly. For obvious reasons, I cannot pre-load every single view of my entire app.

I can reproduce this on your template app in tutorial by commenting out :
  preloads: [
    {
      id: "learn-more"
      location: "example#learn-more"
    }
in /config/structure.coffee

I can provide you a video if you like. By far I have only been using your scanner from app store/google play.

We are all excited about multi page and native navigation between views -  I believe you guys are proud of native performance, you should. But it would be super embarrassing for me to ship an android app with this flickering.

I tried to delete common/assets/loading.html, I got a 404 page for a brief second then the actual page loads (only Androids shows that, iOS is still fine) but no black flickering, I actually thought that's better! But can we disable this loading screen from ... configuration? I cannot find how, would that be a feature request?

I am also accepting any kind of workarounds here, any suggestions? ^*^  
# Kiitos!
